lib/crowbar/client/command/node/list.rb

```diff
@@ -73,6 +73,10 @@ def content_from(request)
                 )
               end
             end
+
+            result.sort do |x, y|
+              x.values.join <=> y.values.join
```

Comment:
I just took a simple method instead of checking if only alias, only name or both are set to decide the sorting based on that.